src/views/tickets/rsvp.php

```diff
@@ -107,17 +107,28 @@ class="tribe-rsvp-message tribe-rsvp-message-error tribe-rsvp-message-confirmati
 				continue;
 			}
 
+			$ticket_id = $ticket->ID;
+
 			/** @var Tribe__Tickets__Tickets_Handler $handler */
 			$handler = tribe( 'tickets.handler' );
 
-			$available = $handler->get_ticket_max_purchase( $ticket->ID );
+			$available = $handler->get_ticket_max_purchase( $ticket_id );
+
```

Comment:
This might be a better place for:

`$readable_amount = tribe_tickets_get_readable_amount( $available, null, false );`

Reply:
```suggestion
			$readable_amount = tribe_tickets_get_readable_amount( $available, null, false );
```